lya_likelihood/py/full_theory.py

```diff
@@ -0,0 +1,192 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import camb_cosmo
+import fit_linP
+import recons_cosmo
+import mean_flux_model
+import thermal_model
+import pressure_model
+import linear_emulator
+import CAMB_model
+
+class FullTheory(object):
+    """Translator between the likelihood object and the emulator. This object
+    will map from a set of CAMB parameters directly to emulator calls, without
+    going through our Delta^2_\star parametrisation """
+
+    def __init__(self,zs,emulator=None,camb_model_fid=None,verbose=False,
+                    mf_model_fid=None,T_model_fid=None,kF_model_fid=None):
+        """Setup object to compute predictions for the 1D power spectrum.
+        Inputs:
+            - zs: redshifts that will be evaluated
+            - emulator: object to interpolate simulated p1d
+            - cosmo_fid: CAMB object with the fiducial cosmology (optional)
+            - verbose: print information, useful to debug."""
+
+        self.verbose=verbose
+        self.zs=zs
+        self.emulator=emulator
+
+        # setup object to compute linear power for any cosmology
```

Comment:
This comment is probably confusing, I don't think this object is ever used to compute anything. I believe this is only used to store the default values for the parameters that we don't vary.